2024/day3/day3.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	_ "embed"
+	"fmt"
+	"regexp"
+	"strconv"
+)
+
+//go:embed input.txt
+var input string
+
+//go:embed test.txt
+var test string
+
+func main() {
+	re := regexp.MustCompile(`mul\((\d+),(\d+)\)`) // matches any pattern with `mul(` and matches any digits
```

Comment:
Problem says X and Y are 1-3 digit numbers this would check for anything more than 1.
![image](https://github.com/user-attachments/assets/ab336eeb-3988-4b39-82bc-819a1558dc03)

Reply:
Good Catch. I wonder how it worked. bt anyways. This one should be fine. `mul\((\d{3}),(\d{3})\)`

Reply:
I think the language says, x and y `are`. So it doesnt have any more than 3 digit number or have the requirement to ignore them. It should be fine i think.